challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -11596,6 +11596,844 @@
+        },
+        {
+          "text":
+            "'<code>isPalindrome(\"ABBA\")</code> should return a boolean.'",
+          "testString":
+            "assert(typeof isPalindrome(\"ABBA\") == 'boolean','<code>isPalindrome(\"ABBA\")</code> should return a boolean.');"
+        },
+        {
+          "text":
+            "'<code>isPalindrome(\"ABBA\")</code> should return <code>true</code>.'",
+          "testString":
+            "assert.equal(isPalindrome(\"ABBA\"),true,'<code>isPalindrome(\"ABBA\")</code> should return <code>true</code>.');"
+        },
+        {
+          "text":
+            "'<code>isPalindrome(\"abcedfgfedcb\")</code> should return <code>false</code>.'",
+          "testString":
+            "assert.equal(isPalindrome(\"abcedfgfedcb\"),false,'<code>isPalindrome(\"abcedfgfedcb\")</code> should return <code>false</code>.');"
+        },
+        {
+          "text":
+            "'<code>isPalindrome(\"abcdeffedcba\")</code> should return <code>true</code>.'",
+          "testString":
+            "assert.equal(isPalindrome(\"abcdeffedcba\"),true,'<code>isPalindrome(\"abcdeffedcba\")</code> should return <code>true</code>.');"
+        },
+        {
+          "text":
+            "'<code>isPalindrome(\"hello\")</code> should return <code>false</code>.'",
+          "testString":
+            "assert.equal(isPalindrome(\"hello\"),false,'<code>isPalindrome(\"hello\")</code> should return <code>false</code>.');"
+        },
+        {
+          "text":
+            "'<code>isPalindrome(\"abcdefgfedcba\")</code> should return <code>true</code>.'",
+          "testString":
+            "assert.equal(isPalindrome(\"abcdefgfedcba\"),true,'<code>isPalindrome(\"abcdefgfedcba\")</code> should return <code>true</code>.');"
+        },
+        {
+          "text":
+            "'<code>isPalindrome(\"ingirumimusnocteetconsumimurigni\")</code> should return <code>true</code>.'",
+          "testString":
+            "assert.equal(isPalindrome(\"ingirumimusnocteetconsumimurigni\"),true,'<code>isPalindrome(\"ingirumimusnocteetconsumimurigni\")</code> should return <code>true</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7f51",
+      "challengeType": 5,
+      "releasedOn": "August 3, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function isPalindrome (str) {", "  // Good luck!", "}"],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Pangram checker",
+      "description": [
+        "A pangram is a sentence that contains all the letters of the English alphabet at least once.",
+        "For example:  <i>The quick brown fox jumps over the lazy dog</i>.",
+        "Write a function to check if a sentence is a  <a href=\"https://en.wikipedia.org/wiki/Pangram\">pangram</a>."
```

Comment:
"Write a function to check if a sentence is a  <a href=\"https://en.wikipedia.org/wiki/Pangram\">pangram</a>. It should return true if the given sentence is a pangram, and false if it isn't."